buildSrc/src/main/kotlin/com/example/chattutorial/Dependencies.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.chattutorial
+
+private const val ANDROID_BUILD_TOOL_VERSION = "4.0.2"
```

Comment:
This is actually the Android Gradle Plugin version, which is not the same as the build tools version.